### ISSUE TITLE
How to set API Key and API Secret to clients? 

### ISSUE THREAD

**Issue by backstageel** [zfcampus/zf-apigility#165]
I want to use Apigility to create an API that several clients will consume.  What I need to do is something like Facebook does: people can create apps that connects to my API,  and I can see what apps are connected. 

Also I would like to be able to limit services for some apps.  Something like app A can online view statuses,  App B can view and post and edit statuses,  App C can only edit statuses for examples.  

Is that possible with apigility? Where can I find documentation for the API Keys and Secret? 

In other words I want application based authorization instead of user based authorization that Oauth2 does.

**Comment by strausmann** [comment]
I have the same question

**Comment by TomHAnderson** [comment]
There is a client_credentials grant type in OAuth2.  If you're asking about scopes I think my api-skeletons/zf-oauth2-doctrine supports them best.

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas-api-tools/api-tools; a new issue has been opened at https://github.com/laminas-api-tools/api-tools/issues/26.
